Client/Blogger/src/utils/Country_data.jsx

```diff
@@ -0,0 +1,253 @@
+const countryList = [
+  "Afghanistan",
```

Comment:
try alternative ways instead having all country list like this, to make code cleaner and easier to manage. and remove the ".jsx" extension as it is not necessary for a file that only exports data

Reply:
@Hareesh-Kumaran , I recommend to use a API for fetching countries list

Reply:
@nagavijaykolla-ces , Yes I was also in search of the API for list of countries but I couldn't find them. Thats why I kept them as a static data.